thankyou_letters/models/partner_communication.py

```diff
@@ -33,6 +33,21 @@ class PartnerCommunication(models.Model):
         domain=[('type', '=', 'sentence')])
     success_sentence = fields.Text(related='success_sentence_id.body_text')
     add_success_story = fields.Boolean(related='config_id.add_success_story')
+    amount = fields.Float(string='Amount', compute='_compute_donation_amount',
+                          store=True)
+
+    @api.multi
```

Comment:
Add a depends tag so that the field is recomputed if the related objects are changes.
`@api.depends('object_ids')`